addOns/websocket/src/main/javahelp/org/zaproxy/zap/extension/websocket/resources/help/contents/pscanrules.html

```diff
@@ -11,5 +11,18 @@ <H1>WebSocket Passive Scan Rules</H1>
         <h2 id="scripts">Scripts</h2>
         Scripts which are included by default in the add-on and they implement the following WebSocket passive scan rules:
 
+        <h3>Information Disclosure: Email address</h3>
+
+        This script scans incoming WebSocket messages for email address. Email address may be not specifically meant for end user consumption.<br>
```

Comment:
"address" > "addresses" (or "address(es)")

Reply:
> Good call, same for the help content.
So we keep this as it is?

Reply:
Hmmm it does read a bit awkwardly. Ummm.... 

How about:
"This script scans incoming WebSocket messages for email addresses. Email addresses may be not specifically meant for end user consumption."

Reply:
Fixed in 944da55ca